src/osd/ReplicatedPG.cc

```diff
@@ -2054,7 +2054,7 @@ void ReplicatedPG::do_op(OpRequestRef& op)
 	}
 
 	ObjectContextRef sobc = get_object_context(clone_oid, false);
-	if (!sobc) {
+	if (!sobc || !sobc->obs.exists) {
```

Comment:
in what operation sobc exists, but sobc->obs.exists is false?

Reply:
This happens after a clone object evicted from the cache tier